basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,247 @@
 const helpers = require('./project-1');
 
-// start testing!
+//========== MULTIPLY BY 10
+describe('multiplyByTen()', () => {
+    it('should multiply provided number by ten', () => {
+        const expected = 50;
+        const actual = helpers.multiplyByTen(5);
+        expect(actual).toEqual(expected);
+    });
+
+    it('should return 0 when called with no value', () => {
+        const product = helpers.multiplyByTen('');
+        expect(product).toBe(0);
+    });
+
+    it('should throw error when called with a string', () => {
+        expect(() => { helpers.multiplyByTen('five') }).toThrow();
+    });
+});
+
+//============ SUBTRACT BY 5
+describe('subtractFive()', () => {
+    it('should subtract provided number by five', () => {
+        const expected = 25;
+        const actual = helpers.subtractFive(30);
+        expect(actual).toEqual(expected);
+    });
+});
+
+//============ SAME LENGTH STRINGS
+describe('areSameLength()', () => {
+    it('should return true if both strings are the same length', () => {
+        const expected = true;
+        const actual = helpers.areSameLength('hello', 'howdy');
+        expect(actual).toEqual(expected);
```

Comment:
Write another test to show that two strings that aren't the same length return `false`.